buildAll.bat

```diff
@@ -11,8 +11,25 @@ call tools\build.bat vioscsi\vioscsi.vcxproj "Win8_SDV Win10_SDV" %*
 if errorlevel 1 goto :fail
 call tools\build.bat viostor\viostor.vcxproj "Win8_SDV Win10_SDV" %*
 if errorlevel 1 goto :fail
+if "%VIRTIO_WIN_SDV_2022%"=="" goto :nosdv2022
+call tools\build.bat Balloon\sys\balloon.vcxproj "Win10_SDV" %*
+if errorlevel 1 goto :fail
+call tools\build.bat fwcfg64\fwcfg.vcxproj "Win10_SDV" %*
+if errorlevel 1 goto :fail
 call tools\build.bat ivshmem\ivshmem.vcxproj "Win10_SDV" %*
 if errorlevel 1 goto :fail
+call tools\build.bat pvpanic\pvpanic\pvpanic.vcxproj "Win10_SDV" %*
+if errorlevel 1 goto :fail
+call tools\build.bat viocrypt\sys\viocrypt.vcxproj "Win10_SDV" %*
+if errorlevel 1 goto :fail
+call tools\build.bat viorng\viorng\viorng.vcxproj "Win10_SDV" %*
+if errorlevel 1 goto :fail
+call tools\build.bat vioserial\sys\vioser.vcxproj "Win10_SDV" %*
+if errorlevel 1 goto :fail
+call tools\build.bat viosock\sys\viosock.vcxproj "Win10_SDV" %*
+if errorlevel 1 goto :fail
```

Comment:
There are missing 3 drivers: viogpu, vioinput and viofs

```
call tools\build.bat viogpu\viogpudo\viogpudo.vcxproj "Win10_SDV" %*
if errorlevel 1 goto :fail
call tools\build.bat vioinput\sys\vioinput.vcxproj "Win10_SDV" %*
if errorlevel 1 goto :fail
call tools\build.bat viofs\pci\viofs.vcxproj "Win10_SDV" %*
if errorlevel 1 goto :fail
```

Reply:
Only viofs is really missing, the rest are skipped for the reason, mentioned above.:)

Reply:
@kostyanf14 we can add it in the next PR.